lib/dialects/mysql/data-types.js

```diff
@@ -60,8 +60,15 @@ module.exports = function (BaseTypes) {
   });
 
   GEOMETRY.parse = GEOMETRY.prototype.parse = function(value) {
+    value = value.buffer();
+
+    //MySQL doesn't support POINT EMPTY, https://dev.mysql.com/worklog/task/?id=2381
+    if(value === null){
```

Comment:
Missing space here

(Really wish this could be handled by the linter @mickhansen , I've noticed you commenting on it more and more :)). JSHint can't do it, dunno if other linters can